Category updates pushed to new branch

andersbarzdukas opened the pull request:
This is the code that adds categorization utilities as well as the baseline selection. Usually there are comments before each function that explain what each function does. 

categorization_utilities.cpp is the file with the new categories and respective functions
zg_functions.cpp has been updated to include some baseline selection functions (not using the bitmap yet)
plot_skeleton.cxx has a sample plotting file with a lot of different new tools highlighted
plot_run2controlregions.cxx is an adaptation of the control region plots made in a previous study (not using a BDT)
plot_run3categories.cxx contains the plots shown for the categorization studies (control region plots to be updated soon)

Changes soon:
- bitmap for baseline selection
- control regions for Run 3 categories
- minor fixes to comments and jet_phi plots in VBF plots

jaebak commented on src/zgamma/plot_peakinghiggsbkgs.cxx:
```diff
@@ -0,0 +1,151 @@
+
+  //Loop containing which plots are made
+  for(unsigned int idx_i(0); idx_i < lep.size(); idx_i++) { 
+    pm.Push<Hist1D>(Axis(80,100,180, "llphoton_m[0]",    "m_{ll#gamma} [GeV]", {122,128}), r2_baseline && lep.at(idx_i), procs, ops).Weight(wgt_1invfb).Tag("ShortName:hpb_");
```
I am having a difficult time understanding what wgt_1invfb is. Where is it defined?

jaebak commented:
Maybe you should try the below commands to see if your code works.
```
git clone git@github.com:richstu/draw_pico.git draw_pico.test
git submodule update --init --remote --recursive
source set_env.sh
git fetch origin pull/2/head:anders
git checkout anders
scons
./run/zgamma/plot_run3categories.exe
```

andersbarzdukas commented:
I forgot that I had changed the samples_zgamma.txt file to be kingscanyon_v0 and added that as well as the those variables used to /txt/variables/pico
I will check now, but that should fix the lingering issues.

andersbarzdukas commented on src/zgamma/plot_peakinghiggsbkgs.cxx:
```diff
@@ -0,0 +1,151 @@
+
+  //Code that pushes the tables
+  pm.min_print_ = true;
+  pm.MakePlots(137.61);
```
I will look at the SetLuminosityTag method then to try and change this with the next push.
